docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  rstudio:
+    image: "ubcdsci/intro-to-ds:v0.6.0-47d5cbf"
```

Comment:
is this docker-compose.yml file auto-generated?

- if so: do we want it to be committed to the repo?
- if not: do we want that `v0.6.0-47blahblah` to be hard-coded?

Reply:
b5c491e and f16fd62 should fix this